srcs/juloo.keyboard2/Keyboard2View.java

```diff
@@ -285,25 +376,38 @@ private int labelColor(KeyValue k, boolean isKeyDown, int defaultColor)
           return _theme.activatedColor;
       }
     }
+    else if (isSubKeyDown)
+    {
+      return _theme.activatedColor;
```

Comment:
"Activated" meant latched in this module. I don't think it's useful to highlighted the keys being pressed specifically.

Reply:
Not very useful but a nice visual feedback

Reply:
I've added this (c175157), I found that I like that the main label is also highlighted.